Formula/mongocli.rb

```diff
@@ -1,8 +1,8 @@
 class Mongocli < Formula
   desc "MongoDB CLI enables you to manage your MongoDB in the Cloud"
   homepage "https://github.com/mongodb/mongocli"
-  url "https://github.com/mongodb/mongocli/archive/refs/tags/mongocli/v1.23.0.tar.gz"
-  sha256 "7b52bf6626cf991b771fc7a62ca7c0ec6deb7012fd5a4c5ef8794b711ee69010"
+  url "https://github.com/mongodb/mongocli/archive/refs/tags/mongocli/v1.23.1.tar.gz"
```

Comment:
We have recently renamed the repository, we should update these two fields with the new name https://github.com/mongodb/mongodb-atlas-cli

Reply:
👍

Reply:
Thank you! : )